event-based-simulators/main/event_based_simulators.py

```diff
@@ -107,17 +107,16 @@ def add_Shiftplan_to_OEE(self):
 
 
     def __create_task(self):   
-        task = PeriodicTask(shiftplan_polling_interval, shiftplan_polling_interval, self.getShiftplan)
-        
-        log.debug(f'create periodic task for pulling shiftplans running every {shiftplan_polling_interval}')      
+        task = PeriodicTask(shiftplan_polling_interval, shiftplan_polling_interval, self.getShiftplan())
+        log.debug(f'Create periodic task for pulling shiftplans running every {shiftplan_polling_interval}')
```

Comment:
However i never saw this output. log is set to info. Is there a way to set it to DEBUG?

Reply:
I think you can change the line number 12 in event_based_simulators.py:
logging.basicConfig(format='%(asctime)s %(name)s:%(message)s', level=logging.INFO)
from INFO to DEBUG and build the package again to see it.

Reply:
Yes but, there should be a way to do it without changing code ❓

Reply:
Since @namnhatpham is on leave i created a Task for it: https://cumulocity.atlassian.net/browse/COE-756